test_python/test_iterators.py

```diff
@@ -0,0 +1,36 @@
+from helper import *
+import pytest
+
+def test_Vector3():
+    double_vec = mrmesh.Vector3d(1, 2, 3)
+    int_vec = mrmesh.Vector3i(1, 2, 3)
+    float_vec = mrmesh.Vector3f(1, 2, 3)
+
+    doubles_list = list(double_vec)
+    ints_list = list(int_vec)
+    floats_list = list(float_vec)
+
+    assert len(doubles_list) == 3
+    assert len(ints_list) == 3
+    assert len(floats_list) == 3
+
+    assert type(doubles_list[0]) == float # Double is converted to float in python
+    assert type(ints_list[0]) == int
+    assert type(floats_list[0]) == float
```

Comment:
please also assert values of list and vec is the same, and also run iteration over `_vec ` to check that it works

Reply:
i've meant something like this for all types
```python
control = 1
for c in double_vec:
    assert c == control
    control = control + 1

Reply:
@Grantim is what I fixed in the last commit what you were looking for or not?

Reply:
Partly, I will update this PR soon